provision/acc_provision/acc_provision.py

```diff
@@ -1628,9 +1651,23 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
 
 
 def generate_apic_config(flavor_opts, config, prov_apic, apic_file):
+    if config.get("target_version"):
```

Comment:
same comments as earlier in L1491-1500, also why can this function not be called from there instead of duplicating the logic in two places?

Reply:
the inline code is different. I will check again